grafana-plugin/src/models/mattermost/mattermost_channel.ts

```diff
@@ -65,6 +92,21 @@ export class MattermostChannelStore extends BaseStore {
     );
   };
 
+  @computed
+  get hasItems() {
+    return Boolean(this.getSearchResult('')?.length);
+  }
+
+  async startAutoUpdate() {
+    this.autoUpdateTimer = setInterval(this.updateMattermostChannels.bind(this), 3000);
+  }
+
+  async stopAutoUpdate() {
+    if (this.autoUpdateTimer) {
+      clearInterval(this.autoUpdateTimer);
+    }
+  }
```

Comment:
Those 3 methods, although I can see they also exist in msteams and telegram, are not used. We can remove them